core/src/style/style.h

```diff
@@ -36,6 +35,14 @@ enum class Blending : int8_t {
     overlay,
 };
 
+struct StyledMesh {
```

Comment:
What made me anxious is that `vboMesh.h` now has to include `style.h`, it would be nice to keep modules with dependencies only within its own module, but this makes sense.

`Drawable` sounds Java, but seems clear enough.